CONTRIBUTING-SCRIPTS.md

```diff
@@ -34,7 +34,7 @@ A checklist for bcc tool development:
 1. **Concise, intuitive, self-explanatory output**. The default output should meet the common need concisely. Leave much less useful fields and data to be shown with options: -v for verbose, etc. Consider including a startup message that's self-explanatory, eg "Tracing block I/O. Output every 1 seconds. Ctrl-C to end.".
 1. **Default output <80 chars wide**. Try hard to keep the output less than 80 characters wide, especially the default output of the tool. That way, the output not only fits on the smallest reasonable terminal, it also fits well in slide decks, blog posts, articles, and printed material, all of which help education and adoption. Publishers of technical books often have templates they require books to conform to: it may not be an option to shrink or narrow the font to fit your output.
 1. **Short tool name**. Follow the style of the other tools, which follow the style of other /usr/bin utilities. They are short and easy to type. No underscores.
-1. **Use pep8 to check Python style**: pep8 --show-source --ignore=E123,E125,E126,E127,E128,E302 filename . Note that it misses some things, like consistent usage, so you'll still need to double check your script.
+1. **Use flake8 to check Python style**: flake8 filename(s) . Note that it misses some things, like consistent usage, so you'll still need to double check your script.
```

Comment:
Does this need to be executed from the root of the repository to use the `.flake8` configuration?

Reply:
http://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations

Reply:
Ok. Then, it might be worth mentioning the command needs to be executed within bcc (peeps might be developing their new tools outside the repo. first). What do you think?

Could you also remove the white space before the dot in that sentence?